### ISSUE TITLE
`alias` keyword

### ISSUE THREAD

**Issue by gwxf0wvr** [IsaacShelton/Adept3x#16]
The commit https://github.com/IsaacShelton/Adept3x/commit/716974cde198896a40ebad4ec228b65ab9cefee0 renamed the `alias` keyword to `typealias`. If there is no longer a standalone `alias` keyword then how can we do function alias?

Btw, I think `type alias` will be better than `typealias`.

**Comment by IsaacShelton** [comment]
Yeah I'm not very satisfied with the `typealias` keyword either right now.

In particular, the syntax of `define`, `alias`, and `func alias` feel like they could be improved, but I'm not sure how yet.

Ideally, the syntax of each would best meet the following criteria:
- Be obvious what it does
- Be similar to similar constructs
- Be brief
- Minimize possibility of conflicting with a likely variable name
- Have the name specified roughly align with names specified by other constructs like `struct`s and `func`s (around 5-7 columns from the left)

Like I mentioned though, I'm not sure what the best way to do that is yet
